.github/workflows/build.yml

```diff
@@ -2,22 +2,24 @@ name: Build and push images
 on:
   push:
     branches:
-    - master
+      - master
     paths:
-    - "*"
-    - "!README.md"
-    - "!build.rb"
+      - '*'
+      - '!README.md'
+      - '!build.rb'
   schedule:
     # As well as running when we make changes we should run at least
     # every week in order to pick up new parent images and new versions of Snyk
-    - cron:  "0 0 * * 0"
+    - cron: '0 0 * * 0'
```

Comment:
Issue: This file is being generated from a template, please add the changes to the template and re-generate this file

Reply:
Suggestion: Can you please add a header comment to say that this file is generated? To avoid issues in the future.